src/btree/bt_debug.c

```diff
@@ -524,11 +547,16 @@ __wt_debug_page(WT_SESSION_IMPL *session, WT_PAGE *page, const char *ofile)
 
 /*
  * __debug_tree --
- *	Dump the in-memory information for a tree.
+ *	Dump the in-memory information for a tree. Takes an explicit btree
+ *	as one may not be set on the session. This is often the case as this
+ *	function will be called from within a debugger, which makes setting a
```

Comment:
This version of the comment reads: 
"Takes an explicit btree as one may not be set on the session."

The other version reads:
"Takes an explicit btree as an argument, as one may not yet be set on the session."

I also don't think the final sentence in the comment adds value - it's obvious from the implementation that it's using ```WT_WITH_BTREE```